docs/internal_api/index.rst

```diff
@@ -143,6 +147,7 @@ Coordinates
    grid.coordinates._populate_lonlat_coord
    grid.coordinates._populate_centroid_coord
    grid.coordinates._construct_xyz_centroids
+   grid.coordinates._set_desired_longitude_range
```

Comment:
Looks like we missed to include some new internal funcs from dataarray_plot.py and grid_plot.py

Reply:
Good catch!

Reply:
Is this addressed yet?

Reply:
It should be now.